README.md

```diff
@@ -43,6 +43,23 @@ docker compose up
 
 - And the Jaeger UI at: <http://localhost:16686>
 
+### Bring your own backend
+
+Likely you want to use the Webstore as a demo application for an observability backend you already have (e.g. an existing instance of jaeger, zipkin or one of the [vendor of your choice](https://opentelemetry.io/vendors/). Adding this backend is easy: Before running docker compose, open the [otelccol-config.yml](./src/otelcollector/otelcol-config.yml) with an editor and add an trace exporter for your backend, e.g.:
```

Comment:
maybe - make it clear that this is only applicable if the vendor has native OTLP ingesting endpoint.
And mention that, some vendor backends requires separate exporters, available in collector-contrib repos.

Reply:
makes sense, thank you. Updated!